client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -37,6 +38,13 @@ const PatternAssembler: Step = ( { navigation } ) => {
 	const siteId = useSiteIdParam();
 	const siteSlugOrId = siteSlug ? siteSlug : siteId;
 
+	useEffect( () => {
+		// Require to start the flow from the first step
+		if ( ! selectedDesign ) {
+			location.href = `/setup${ location.search }`;
+		}
+	} );
```

Comment:
We can use 
```
useEffect( () => {
		// Require to start the flow from the first step
		if ( ! selectedDesign ) {
			location.href = `/setup${ location.search }`;
		}
	}, [] );
```

To make sure it's only run once.

Reply:
Okay, I added it. 

I think it would run only once anyways because of the redirection. I didn't add it initially to avoid this warning:
> React Hook useEffect has missing dependencies: 'goToStep' and 'selectedDesign'. Either include them or remove the dependency array.